tests/foreman/ui/test_myaccount.py

```diff
@@ -199,22 +176,27 @@ def test_positive_update_password(self):
         """
         for password in _valid_string_data(max_len=254):
             with self.subTest(password):
-                with self.logged_test_user():
-                    self.my_account.navigate_to_entity()
-                    self.my_account.wait_until_element(
-                        locators['users.password'])
-                    self.my_account.assign_value(
-                        locators['users.password'], password)
-                    self.my_account.assign_value(
-                        locators['users.password_confirmation'], password)
-                    self.my_account.click(common_locators['submit'])
-                    self.my_account.wait_until_element_is_not_visible(
-                        common_locators["notif.success"])
-                    self.login.logout()
-                    self.login.login(self.account_user.login, password)
+                with Session(self.browser, self.account_user.login,
```

Comment:
@renzo think a more simple approach will be to create a new user for each iteration and simply verify that he can login with the new password (same logic as with language) and delete it at the end.
Because if it fail in the middle (on line 186) for one of the sub tests, the other sub tests will not have a chance to succeeded as the password is not reverted back and the user cannot login to change the password in the beginning.

I think you had a similar approach in earlier versions but only with one iteration, or I am wrong (simply cannot find the code).